engine/datamanager.go

```diff
@@ -1345,6 +1338,9 @@ func (dm *DataManager) GetTrend(tenant, id string,
 			return
 		}
 	}
+	if err := tr.uncompress(dm.ms); err != nil {
```

Comment:
Here you do not need to redefine err since it is already defined in the function's signature.